tests/nuget_versioning_tests.README

```diff
@@ -0,0 +1,32 @@
+﻿![NuGet logo](https://raw.githubusercontent.com/NuGet/Home/dev/meta/resources/nuget.png)
```

Comment:
This should be saved with an ".md" extension, it can't be previewed right now

Reply:
Fixed. Thanks!